tests/console/console.go

```diff
@@ -129,6 +130,36 @@ func RunCommand(vmi *v1.VirtualMachineInstance, command string, timeout time.Dur
 	return nil
 }
 
+// RunCommandAndStoreOutput runs the command line from `command` connecting to an already logged in console in vmi.
+// The output of `command` is returned as a string
+func RunCommandAndStoreOutput(vmi *v1.VirtualMachineInstance, command string, timeout time.Duration) (string, error) {
```

Comment:
This seems to be very similar to the existing `SafeExpectBatchWithResponse` function.
Is there a specific reason to have this additional one with all this login duplicated?

I suspect this introduces a potential flakiness due to the missing checks (that we do have a console, that the command did reach the console, etc).